apps/web/src/routes/_content/resources/podcasts/$podcastId.tsx

```diff
@@ -23,15 +32,20 @@ function Podcast() {
     id: Number(podcastId),
     language: i18n.language ?? 'en',
   });
+
+  const { data: suggestedPodcasts, isFetched: isFetchedSuggested } =
+    trpc.content.getPodcasts.useQuery({
+      language: i18n.language ?? 'en',
+    });
   const isScreenMd = useGreater('sm');
 
   function displayAbstract() {
```

Comment:
This part is reused in both book and podcast -> should be a single reusable component to prevent redundancy